### DISCUSSION TITLE
Can't see how to edit HTML in the rich text/editor type

### DISCUSSION THREAD

**Discussion by NafunaAfrica** [pocketbase/pocketbase#5545] [Q&A]
I am wriing a blog and need to embed a video from youtube in the post. I used to be able to do it but now i cant seem to find option in the editor anymore... very confusing

**Comment by ganigeorgiev** [comment]
It is more confusing reading posts like this.

There are no changes to the `editor` field or its options in the recent releases so I'm not sure to what exactly do you refer to that was available before but now it's missing.

I don't think that we ever had a quick button/control for adding embedded content in the text editor and for now this could be done by pasting the embed code through the "Source code" panel (_second to last button_).

Note that external embedded resources (YouTube, Vimeo, etc.) will not be visualized in the UI for security reasons since they will conflict with the default Conent Security Policy (CSP) of the dashboard (they will be still available in the response but most likely displayed as blank rectangle in the editor preview).

With that said, please don't treat the issues tracker and discussions as llm chat and try to more clearly elaborate what actually is the issue and what you are trying to do.

**Reply by NafunaAfrica** [reply]
![image](https://github.com/user-attachments/assets/f6019b1e-4594-44a7-b03e-192e3ee623e4)
I was pretty sure this kind of thing would be reported. I dont see how I treated this like LLM chat? Here is the screenshot, maybe shine your ray of wisdom there and explain that?

**Reply by ganigeorgiev** [reply]
You are either using a very old PocketBase version (_check in the bottom-right footer_) or there is a browser extension that interfere with the JS of the editor. This is how it should look like from the live demo:
![editor_screenshot](https://i.imgur.com/S56uaug.png)

Adding embeded code at the moment could be done via the "Source code" button.

>  I dont see how I treated this like LLM chat? 

By just reporting one confusing sentence without any further details. This is a volunteer driven project and I work on it on my free time. Neither you, nor anyone else is entitled to anything more than what is already available.

If you are not satisfied with something, submit a feedback, elaborate how and what you want something to be, but claiming "... very confusing" is just an opinion. You are free to use something else if you want.
